notebooks/supernovae_plus_extended_source_tutorial.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<p>can you document these inputs directly in the notebook? Also, can you give a description of how to download the scotch catalogue and have a more modular path (stating where you would need to put the file</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>